index.html

```diff
@@ -61,7 +61,7 @@ <h4>Stable</h4>
     <div class="row">
       <div class="col-sm-12">
         <a id="asf-current-event-logo" href="https://www.apache.org/events/current-event.html"><img alt="ASF Current Events" class="img-responsive center-block" src="https://www.apache.org/events/current-event-234x60.png"/></a><br/>
-        <a id="accumulo-summit-logo" href="http://accumulosummit.com/"><img alt="Accumulo Summit" class="img-responsive center-block" src="{{ site.baseurl }}/images/accumulo-summit-2017-logo-horizontal.png"/></a>
+        <a id="accumulo-summit-logo" href="http://accumulosummit.com/"><img alt="Accumulo Summit" class="img-responsive center-block" src="{{ site.baseurl }}/images/accumulo-summit-logo.png"/></a>
```

Comment:
It is that image.  I just renamed the file.

Reply:
Or are you suggesting we just link to the image?

Reply:
Hmm, that's weird. Last time, I only saw the old image in the GitHub PR. Now I see you're removing one and replacing it with the other. The image is fine. I must've clicked on something incorrectly.

I'm not sure how linking will work, since their site doesn't support https. Linking to it may give a browser warning. There's a trend in browsers to push publishers to use https by giving big warnings for mixed content or http content.